src/components/riskmap/AccidentLocationList.tsx

```diff
@@ -63,7 +63,11 @@ const AccidentLocationList: React.FC<AccidentLocationListProps> = ({
                 ? location["Laatste eindtijd"].toTimeString()
                 : ""
             }
-            color={location["Points"] ? "orange" : "red"}
+            color={
+              location["Hmp tot"] && !isNaN(location["Hmp tot"])
```

Comment:
Can't this check be simplified as `location["Hmp tot"]` as NaN is considered a falsy value that will make the first check fail already.